paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -305,6 +303,18 @@ class Paparazzi @JvmOverloads constructor(
           lifecycleOwner.registry.currentState = Lifecycle.State.RESUMED
         }
 
+        if (hasComposeRuntime) {
+          // During onAttachedToWindow, AbstractComposeView will attempt to resolve its parent's
+          // CompositionContext, which requires first finding the "content view", then using that
+          // to find a root view with a ViewTreeLifecycleOwner
+          viewGroup.id = android.R.id.content
+
+          // ComposeUi uses it's own `AndroidUiDispatcher` by default. Unfortunately this dispatcher
+          // does not implement delay which causes ComposeUi to use DefaultDelay which will be run
+          // async to the test. By initializing the Recomposer with Dispatchers.Main delay will be backed by our test Handler.
+          modifiedView.compositionContext = modifiedView.createLifecycleAwareWindowRecomposer(Dispatchers.Main)
```

Comment:
`createLifecycleAwareWindowRecomposer` requires LifecycleOwner to be already installed so moving this hasComposeRuntime block below that setup.